components/ModSubmissionListContainer.vue

```diff
@@ -33,7 +33,10 @@
                     @click="handleClickToSubmissionForm(submission.id)"
                 >
                     <span class="text-start">{{ index + 1 }}</span>
-                    <span class="text-start">{{ submission.healthcareProfessionalName }}</span>
+                    <span
+                        :data-testid="`mod-submission-name-${submission.healthcareProfessionalName}`"
```

Comment:
What do you think about using either the index or an ID here instead of the `heathcareProfessionalName`? An `id` would be more uniquely identifiable and probably easier to test for. It's possible for multiple healthcare professionals to have the same name.

Perhaps we could do something like: 
```
:data-testid="`mod-submission-healthcareprofessional-${submission.id}`"
```